client/python/README.md

```diff
@@ -1,3 +0,0 @@
-This folder contains ModelDB's
```

Comment:
I migrated this to the .rst. PyPI can render .rst files, but not .md, and it seemed redundant to have both. Github will display either .rst or .md in the file folder, but not both.